tests/converter/test_code_generation.py

```diff
@@ -336,9 +336,18 @@ def codegen_cpu(self,
                cache_dma=cache_dma,
                )
 
-        lib_name = 'libdlk_' + cpu_name
+        cpu_to_lib = {
+            'aarch64_False':'aarch64', 
+            'arm_False':'arm',
+            'arm_fpga_False':'fpga', 
+            'x86_64_False':'x86',
+            'x86_64_True':'x86_avx'
+        }
+
+        lib_base_name = cpu_to_lib[cpu_name + '_' + str(use_avx)]
```

Comment:
AVX is for x86 only.
Added True and False flag to all CPU architecture looks redundant.

Reply:
Yes, I know AVX flag is redundant for other CPU than x86.
But I used AVX flag for all CPUs to apply `cpu_to_lib`  table to make the code simple.

Is `if statement` better than translation table ?

Reply:
> But I used AVX flag for all CPUs to apply cpu_to_lib table to make the code simple.

Sorry, If my understand of your code is correct, I do not agree with you...
Create new table to just add `_avx` looks not suitable way.

How about to write like

```
lib_base_name = "x86_avx" if use_avx else cpu_name
```

Reply:
I wanted to remove `if statement` but I gave it up .
I changed to use `if statement` instead of table.
Is it OK for you ?